packages/autocomplete-plugin-algolia-insights/src/__tests__/createAlgoliaInsightsPlugin.test.ts

```diff
@@ -79,6 +86,88 @@ describe('createAlgoliaInsightsPlugin', () => {
     );
   });
 
+  test('sets clickAnalytics: true for getAlgoliaResults and getAlgoliaFacets', async () => {
+    const insightsClient = jest.fn();
+    const insightsPlugin = createAlgoliaInsightsPlugin({ insightsClient });
+
+    const searchClient = createSearchClient({
+      search: jest.fn(() =>
+        Promise.resolve(
+          createMultiSearchResponse<{ label: string }>(
+            {
+              hits: [{ objectID: '1', label: 'Hit 1' }],
+            },
+            {
+              facetHits: [{ count: 2, value: 'Hit 2' }],
+            }
+          )
+        )
+      ),
+    });
+
+    const playground = createPlayground(createAutocomplete, {
+      plugins: [insightsPlugin],
+      getSources({ query }) {
+        return [
+          {
+            sourceId: 'hits',
+            getItems() {
+              return getAlgoliaResults({
+                searchClient,
+                queries: [
+                  {
+                    indexName: 'indexName',
+                    query,
+                  },
+                ],
+              });
+            },
+            templates: {
+              item({ item }) {
+                return JSON.stringify(item);
+              },
+            },
+          },
+          {
+            sourceId: 'facets',
+            getItems() {
+              return getAlgoliaFacets({
+                searchClient,
+                queries: [
+                  {
+                    indexName: 'indexName',
+                    facet: 'categories',
+                    params: {
+                      facetQuery: query,
+                    },
+                  },
+                ],
+              });
+            },
+            templates: {
+              item({ item }) {
+                return JSON.stringify(item);
+              },
+            },
+          },
+        ];
+      },
+    });
+
+    userEvent.type(playground.inputElement, 'a');
+    await runAllMicroTasks();
+
+    expect(searchClient.search).toHaveBeenCalledTimes(1);
+    expect(searchClient.search).toHaveBeenCalledWith([
+      expect.objectContaining({
+        params: expect.objectContaining({ clickAnalytics: true }),
+      }),
+      expect.objectContaining({
+        params: expect.objectContaining({ clickAnalytics: true }),
```

Comment:
is this behaviour expected (facet search with clickAnalytics: true)

Reply:
looks like no queryID is returned anyway, it's likely ignored by the engine

Reply:
Do we want to filter it out to avoid sending an unnecessary larger payload?

Reply:
to prevent wrong clickAnalytics, you don't want to say the search is tracked, when there's no clicks happen

Reply:
We discussed and this is unlikely, and we'll deal with it when there ever is a queryID in sffv